src/Compilers/Core/Portable/Operations/ICollectionElementInitializerOperation.cs

```diff
@@ -1,23 +1,23 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Immutable;
 
 namespace Microsoft.CodeAnalysis.Operations
 {
     /// <summary>
-    /// Represents a collection element initializer.
+    /// Obsolete interface that used to represent a collection element initializer. It has been replaced by
+    /// <see cref="IInvocationOperation"/> and <see cref="IDynamicInvocationOperation"/>, as appropriate.
     /// <para>
     /// Current usage:
-    ///  (1) C# nested collection element initializer expression within a collection initializer.
-    ///      For example, given a collection initializer "new Class() { Y = { { x, y, 3 } } }",
-    ///      nested collection element initializer for Y, i.e. "{ { x, y, 3 } }" is represented by this operation.
-    ///  (2) VB nested collection element initializer expression within a collection initializer.
+    ///   None. This API has been obsoleted in favor of <see cref="IInvocationOperation"/> and <see cref="IDynamicInvocationOperation"/>.
     /// </para>
     /// </summary>
     /// <remarks>
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
+    [Obsolete("ICollectionElementInitializerOperation has been replaced with IInvocationOperation and IDynamicInvocationOperation", error: true)]
```

Comment:
>has [](start = 54, length = 3)

Use nameof here?

Reply:
Used nameof everywhere I could. I couldn't use nameof(ICollectionElementInitializer) in the attributes on the visitors/OperationKind, so I kept the plain text there but still swapped IInvocationOperation and IDynamicInvocationOperation for nameof.

---
In reply to: [190395155](https://github.com/dotnet/roslyn/pull/26781#discussion_r190395155) [](ancestors = 190395155)